egs/zeroth_korean/s5/local/chain/multi_condition/run_tdnn_1n.sh

```diff
@@ -0,0 +1,302 @@
+#!/bin/bash
+
+set -e -o pipefail
```

Comment:
again- names should be consecutive starting from 1a.n   And you should include results in a comment at the top of the script, preferably the output of some script like local/chain/compare_wer.sh.